charts/crowdsec/values.schema.json

```diff
@@ -815,6 +818,37 @@
                 }
             },
             "required": ["source", "url", "query"]
+        },
+        "KubernetesSource": {
+            "type": "object",
+            "description": "https://docs.crowdsec.net/docs/next/log_processor/data_sources/kubernetes_audit/",
+            "additionalProperties": false,
+            "properties": {
+                "source": {
+                    "const": "k8s-audit"
+                },
+                "labels": {
+                    "$ref": "#/definitions/Labels"
+                },
+                "listen_addr": {
+                    "type": "string",
+                    "default": "127.0.0.1",
+                    "description": "Address on which the syslog will listen."
+                },
+                "listen_port": {
+                    "type": "number",
+                    "default": 9876,
+                    "description": "The port on which the datasource will listen."
+                },
+                "webhook_path": {
+                    "type": "string",
+                    "default": "/webhook",
+                    "description": "The path on which the datasource will accept requests from kubernetes."
+                }
+            },
+            "required": [
+                "source"
```

Comment:
All the fields except `labels` should be added in the required as they are all mandatory.

Reply:
Added in commit https://github.com/crowdsecurity/helm-charts/pull/216/commits/22017d55cf202efa8904cbad0520401b914c01e2